src/uncrustify_types.h

```diff
@@ -310,8 +310,8 @@ enum lang_flag_e
    LANG_PAWN = 0x0080,
    LANG_ECMA = 0x0100,     //! ECMA Script (JavaScript)
 
-   LANG_ALLC = 0x017f,     /** LANG_C    | LANG_CPP | LANG_D    | LANG_CS   |
-                            *  LANG_JAVA | LANG_OC  | LANG_VALA | LANG_ECMA   */
+   LANG_ALLC = 0x017f,     //  LANG_C    | LANG_CPP | LANG_D    | LANG_CS   |
```

Comment:
Uh, this seems to have changed from a Doxygen to non-Doxygen comment. Maybe you wanted `//!`?

Also, since this is being changed, it looks like the `FLAG_HDR` comment should also be made consistent?